src/main/resources/matlab-bamboo-plugin.properties

```diff
@@ -33,6 +33,13 @@ matlab.test.coverage.exists= HTML code coverage
 matlab.model.coverage.exists= HTML model coverage
 matlab.code.coverage.directory= Folder path
 
+#Test run customizations
+matlab.test.customizations=Customize test run
+matlab.test.strict=Strict
+matlab.test.use.parallel=Use parallel
+matlab.test.output.detail=Output Detail
+matlab.test.logging.level=Logging level
```

Comment:
"Output detail" (lower case D) to be consistent with the other options

Reply:
Thanks for pointing it out, updated!